packages/@o3r/logger/src/services/logger/logger.spec.ts

```diff
@@ -60,18 +60,37 @@ describe('Logger service', () => {
   describe('with third-party client', () => {
     let service: LoggerService;
     let client: LoggerClient;
+    const identifyMock = jest.fn();
+    const eventMock = jest.fn();
+    const getSessionURLMock = jest.fn();
+    const stopRecordingMock = jest.fn();
+    const resumeRecordingMock = jest.fn();
+    const errorMock = jest.fn();
+    const warnMock = jest.fn();
+    const logMock = jest.fn();
+    const createMetaReducerMock = jest.fn();
 
     beforeEach(() => {
+      identifyMock.mockReset();
+      eventMock.mockReset();
+      getSessionURLMock.mockReset();
+      stopRecordingMock.mockReset();
+      resumeRecordingMock.mockReset();
+      errorMock.mockReset();
+      warnMock.mockReset();
+      logMock.mockReset();
+      createMetaReducerMock.mockReset();
+      getSessionURLMock.mockReturnValue('client.session.url');
```

Comment:
that seems like a lot of boilerplate just to please the linter

Reply:
no need of changes with `jest/unbound-method`